src/simulation.h

```diff
@@ -17,22 +22,22 @@ typedef struct chunk {
 // always freed as part of a simulation history
 Chunk *new_chunk();
 
-typedef struct simulationHistory {
+typedef struct simulation_history {
   Chunk *first_chunk;
   Chunk *last_chunk;
 } SimulationHistory;
 
 SimulationHistory *new_simulation_history();
 void free_simulation_history(SimulationHistory *simulation_historyp);
-void insert_reaction(SimulationHistory *simulation_historyp, int reaction);
+void insert(SimulationHistory *simulation_historyp, int reaction, double time);
```

Comment:
insert_history_element is better than insert here because C is an antiquated langauge with no namespaces